<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltrWidget</class>
 <widget class="QWidget" name="FiltrWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonExit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStart">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать Фильтрацию</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelTextValue">
      <property name="text">
       <string>Значение для фильтрации: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineValue"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
